subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -1047,33 +971,40 @@ The main difference between a _dynamic version_ and a _changing module_ is that
 
 By default, Gradle caches dynamic versions and changing modules for 24 hours. You can override the default cache modes using <<sec:cache_command_line_options,command line options>>. You can change the cache expiry times in your build using the resolution strategy (see <<sec:controlling_caching>>).
 
-[[sec:strategies_of_transitive_dependency_management]]
-=== Strategies for transitive dependency management
+[[sec:controlling_caching]]
```

Comment:
Nitpick about the anchor here. For our own maintenance, it'd be nice if it was clear where I could find this section. `sec:controlling_dependency_caching` would be better.

Reply:
done